### ISSUE TITLE
View Code window is not respomsive 

### ISSUE THREAD

**Issue by morpheus-30** [foss42/api-dash#67]
**Describe the bug/problem**
While opening the ViewCode Sidebar, if we reduce the width of the whole window, Pixels are overflowed

**Steps to Reproduce the bug/problem**
>Open API Dash
>Click on View Code
>Reduce the width of API Dash window

**Expected behavior**
It must be responsive

**Device Info (The device where you encountered this issue):**
 - OS: LINUX
 - Version: Ubuntu 22.04.2 LTS
![Screenshot from 2023-06-23 17-24-53](https://github.com/foss42/api-dash/assets/95480625/07016bee-05f2-45c2-9cc9-516dc21574b1)

**Comment by animator** [comment]
Hi @morpheus-30 
Thank you for reporting this bug. In case you have a fix in mind and want to send across a PR please feel free to mention your solution below.

**Comment by Saimurugeshwari** [comment]
can I work on this issue under GSSOC23

**Comment by Saimurugeshwari** [comment]
https://blog.logrocket.com/create-responsive-layouts-flexible-expanded-widgets-flutter/

using flexiable and expand we can make the window resposive.

**Comment by morpheus-30** [comment]
I fixed the issue, just need to open the pull request :)

**Comment by Saimurugeshwari** [comment]
okay. Thank you for the response

**Comment by animator** [comment]
@morpheus-30 please open a new PR for this issue.
@Saimurugeshwari we have plenty of issues to work on under GSSoC. Do check them out (and read https://github.com/foss42/api-dash/issues/56). Thanks!

**Comment by Saimurugeshwari** [comment]
@animator Thank you, I will follow that

**Comment by morpheus-30** [comment]
@animator I am kinda stuck in git, I opened a pull request for the code gen, and in the same branch I fixed the responsive issue, how can I open another pull request for it in this case? Do I need to make a new branch?

**Comment by animator** [comment]
@morpheus-30 Yes, you need to make a new branch for responsive issue fix and send across the PR.

**Comment by animator** [comment]
Issue has been fixed in https://github.com/foss42/api-dash/commit/2276fcfe1a66474f40f46b3893465b3979bc5a8c
